qutip/solver/options.py

```diff
@@ -0,0 +1,215 @@
+__all__ = ['SolverOptions',
+           'SolverResultsOptions', 'SolverOdeOptions',
+           'McOptions']
+
+from ..optionsclass import optionsclass
+import multiprocessing
+
+@optionsclass("solver")
+class SolverOptions:
+    """
+    Class of options for evolution solvers such as :func:`qutip.mesolve` and
+    :func:`qutip.mcsolve`. Options can be specified either as arguments to the
+    constructor::
+
+        opts = SolverOptions(order=10, ...)
+
+    or by changing the class attributes after creation::
+
+        opts = SolverOptions()
+        opts.order = 10
+
+    Returns options class to be used as options in evolution solvers.
+
+    The default can be changed by::
+
+        qutip.settings.solver['order'] = 10
+
+    Options
+    -------
+    """
+    options = {
+        # (turned off for batch unitary propagator mode)
+        "progress_bar": "text",
+        # Normalize output of solvers
+        # (turned off for batch unitary propagator mode)
+        "progress_kwargs": {"chunk_size":10},
+    }
+
+
+@optionsclass("ode", SolverOptions)
+class SolverOdeOptions:
+    """
+    Class of options for evolution solvers such as :func:`qutip.mesolve` and
+    :func:`qutip.mcsolve`. Options can be specified either as arguments to the
+    constructor::
+
+        opts = SolverOptions(order=10, ...)
```

Comment:
`SolverOptions` here should be `SolverOdeOptions` but the rest of the opening parts of the docstrings for the two clases should also say something different.